core/controllers/topics_and_skills_dashboard.py

```diff
@@ -294,27 +346,60 @@ def post(self):
 class NewSkillHandler(base.BaseHandler):
     """Creates a new skill."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'description': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_nonempty'
+                    }, {
+                        'id': 'has_length_at_most',
+                        'max_value': (
+                            android_validation_constants
+                            .MAX_CHARS_IN_SKILL_DESCRIPTION
+                        )
+                    }]
+                }
+            },
```

Comment:
Do we still need `skill_domain.Skill.require_valid_description(description)` down below?

Reply:
removed it